setup.py

```diff
@@ -64,7 +66,9 @@ def readme():
           'h5py>=2.6',
           'websockets>=3.2',
           'jsonschema',
-          'pyzmq'
+          'pymongo',
```

Comment:
pymongo be gone

Reply:
...waiting for it to be gone....

Reply:
How did you set up this bot @astafan8 ? ;-)